File: outros/palindromo.js
// Função q verifica se o texto inserido é um palíndromo

function palindromo(texto) {
	for (let i = 0; i < texto.length; i++) if (texto[i] != texto[texto.length - 1 - i]) return false;
	return true;
}

console.log(palindromo("luzazul"));

File: outros/c/palindromo.c
#include <stdio.h>

int palindromo(char string[20])
{
  int limit = 0;
  for (int i = 0; string[i] != '\0'; i++)
    limit++;
  for (int j = 0; j < limit; j++) {
    if (string[j] != string[limit - j - 1])
      return 1;
  }
  return 0;
}

int main()
{
  char palavra[20];
  puts("Programa que verifica se uma string é um palíndromo");
  puts("Insira a palavra: ");
  scanf("%s", palavra);
  int result = palindromo(palavra);
  if (result)
  puts("Não é um palíndromo");
  else
  puts("É um palíndromo");
  return 0;
}
